create.ps1

```diff
@@ -195,17 +195,10 @@ try {
     Write-Verbose 'Getting instance url'
     $instanceUri = $($config.baseUrl)
 
-    Write-Verbose 'Getting total number of users'
-    $response = Invoke-ScimRestMethod -InstanceUri $instanceUri -Endpoint 'Users' -Method 'GET' -headers $headers
-    $totalResults = $response.totalResults
-
-    Write-Verbose "Retrieving '$totalResults' users"
-    $responseAllUsers = Invoke-ScimRestMethod -InstanceUri $instanceUri -Endpoint 'Users' -Method 'GET' -headers $headers -TotalResults $totalResults
-
-    Write-Verbose "Verifying if account for '$($p.DisplayName)' must be created or correlated"
-    $lookup = $responseAllUsers | Group-Object -Property 'userName' -AsHashTable
-    $userObject = $lookup[$account.UserName]
-    if ($userObject){
+    Write-Verbose "Checking if user [$($p.DisplayName)] exists"

```

Comment:
MIsschien goed om hier ook nog de gebruikers naam te melden, omdat dat het veld is waarmee het account wordt gezocht?